README.md

```diff
@@ -49,6 +49,9 @@ Arguments:
   --since         -- if you only want to pull claims data added after a certain date.
                      The expected format is yyyy-MM-dd'T'HH:mm:ss.SSSXXX+/-ZZ:ZZ.
                      Example March 1, 2020 at 3 PM EST -> 2020-03-01T15:00:00.000-05:00. More below.
+  --until         -- if you only want to pull claims data added before a certain date.
```

Comment:
is it claims data added before a certain date, or claims data updated before a certain date? 

I also saw you use updated or filed in another reference, i'm assuming that's the same thing?